src/constants/tokenSafety.tsx

```diff
@@ -58,24 +58,28 @@ export type Warning = {
   message: JSX.Element
   /** Determines whether triangle/slash alert icon is used, and whether this token is supported/able to be traded. */
   canProceed: boolean
+  displayLabel: boolean
 }
 
 const MediumWarning: Warning = {
   level: WARNING_LEVEL.MEDIUM,
   message: <Trans>Caution</Trans>,
   canProceed: true,
+  displayLabel: false,
```

Comment:
if i am reading the description correctly, this is determined by the level of the warning right? so i think an easier way to do this would be create a function to return whether to display the label based on the level. that makes it more clear to other people the reasoning is based on the level, and this isn't an "independent variable"

Reply:
Added function